api-guidelines/rest/http/headers/rules/may-use-test-header.md

```diff
@@ -0,0 +1,44 @@
+---
+id: R000080
+---
+
+# MAY use `Test` header
+
+API consumers and providers may use the OTTO-specific `Test` header described within this rule to communicate that a request or response is part of a test.
+
+Test requests are created in various contexts.
+Some of them require special handling on provider or consumer side, such as bypassing specific validations or using modified business logic while processing.
+This rule defines the test header that API consumers and providers can use to identify test data.
+Downstream teams must then forward this test header in subsequent processing steps, e.g. as a test extension in an asynchronous event.
```

Comment:
...in an asynchronous event. > isn't this rule for REST? Shouldn't this read synchronous then?

Reply:
Yes, but it's possible that API providers sending async events as an direct consequence of a synchronous API call, e.g. place order REST api triggers a "order placed" event. In this case it's also important to include the original test headers in the event.